packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7244,4 +7247,55 @@ export default class Meeting extends StatelessWebexPlugin {
 
     await Promise.all(unpublishPromises);
   }
+
+  /**
+   * Touch enable breakout resource
+   * return breakout resource info
+   * @returns {Promise}
+   */
+  private async touchBreakout() {
+    const breakoutUrl = `${this.breakoutServiceUrl}/breakout/`;
+
+    await this.meetingRequest
+      .touchBreakout({breakoutUrl, locusUrl: this.locusUrl})
+      .then((response) => {
+        if (response && response.body && response.body.breakout) {
+          this.breakoutInfo = response.body;
+        }
+      })
+      .catch((err) => {
+        LoggerProxy.logger.error(
+          `Meeting:index#touchBreakout. --> can't get breakout resource ${err}`
+        );
+        throw err;
+      });
+  }
+
+  /**
+   * Method to enable or disable the meeting breakout session
+   * @param  {boolean} enable - enable or disable breakout
+   * @returns {Promise}
+   * @public
+   * @memberof Meeting
+   */
+  public async toggleBreakout(enable: boolean) {
+    this.breakoutInfo = this.locusInfo?.controls?.breakout;
+
+    if (!this.breakoutInfo) {
+      await new Promise((resolve, reject) => {
+        this.touchBreakout();
+      });
+    }
+
+    if (this.breakoutInfo && this.breakoutInfo.url) {
```

Comment:
do a .then from previous touchBreakOut then add the below logic

Reply:
OK, got it, thanks